demos/fleet_provisioning/fleet_provisioning_with_csr/claim_policy.json

```
@@ -0,0 +1,31 @@
+ {
+   "Version": "2012-10-17",
+   "Statement": [
+     {
+       "Effect": "Allow",
+       "Action": [
+         "iot:Connect"
+       ],
+       "Resource": "*"
+     },
+     {
+       "Effect": "Allow",
+       "Action": [
+         "iot:Publish",
+         "iot:Receive"
+       ],
+       "Resource": [
+         "arn:aws:iot:us-west-2:012345678901:topic/$aws/certificates/create-from-csr/*",
```

We shouldn't be providing any AWS account information in the examples. The policy should carry templatized information.
```suggestion
         "arn:aws:iot:<aws-region>:<account-id>:topic/$aws/certificates/create-from-csr/*",
```